CONTRIBUTING.md

```diff
@@ -1,116 +1,3 @@
-We also use the `__init__.py` file as a place to persist the version data for occaissions where the `git` history is unavailable or unwanted.
-
-The following command is used to add `poetry-dynamic-versioning` to Poetry for use with this project: `poetry self add "poetry-dynamic-versioning[plugin]"`.
-Versioning for the project is intended to align with GitHub Releases which provide `git` tag capabilities.
-
-### Releases
-
-We publish source code by using [GitHub Releases](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases) available [here](https://github.com/wayscience/cosmicqc/releases).
-We publish a related Python package through the [Python Packaging Index (PyPI)](https://pypi.org/) available [here](https://pypi.org/project/cosmicqc/).
-
-#### Release Publishing Process
-
-Several manual and automated steps are involved with publishing coSMicQC releases.
-See below for an overview of how this works.
-
-Notes about [semantic version](https://en.wikipedia.org/wiki/Software_versioning#Semantic_versioning) (semver) specifications:
-coSMicQC version specifications are controlled through [`poetry-dynamic-versioning`](https://github.com/mtkennerly/poetry-dynamic-versioning) which leverages [`dunamai`](https://github.com/mtkennerly/dunamai) to create version data based on [git tags](https://git-scm.com/book/en/v2/Git-Basics-Tagging) and commits.
-coSMicQC release git tags are automatically applied through [GitHub Releases](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases) and related inferred changes from [`release-drafter`](https://github.com/release-drafter/release-drafter).
-
-1. Open a pull request and use a repository label for `release-<semver release type>` to label the pull request for visibility with [`release-drafter`](https://github.com/release-drafter/release-drafter) (for example, see [coSMicQC#24](https://github.com/wayscience/cosmicqc/pull/24) as a reference of a semver patch update).
-1. On merging the pull request for the release, a [GitHub Actions workflow](https://docs.github.com/en/actions/using-workflows) defined in `draft-release.yml` leveraging [`release-drafter`](https://github.com/release-drafter/release-drafter) will draft a release for maintainers.
-1. The draft GitHub release will include a version tag based on the GitHub PR label applied and `release-drafter`.
-1. Make modifications as necessary to the draft GitHub release, then publish the release (the draft release does not normally need additional modifications).
-1. On publishing the release, another GitHub Actions workflow defined in `publish-pypi.yml` will run to build and deploy the Python package to PyPI (utilizing the earlier modified `pyproject.toml` semantic version reference for labeling the release).
+Please see our [contributing](https://WayScience.github.io/coSMicQC/main/contributing) documentation for more details on contributions, development, and testing.
```

Comment:
Same 404 error comment here

Reply:
Thanks - yes, won't be available until after this PR is merged.